osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -0,0 +1,169 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package packagemanagement
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	osconfigpb "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha1"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/compute"
+	osconfigserver "github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/osconfig_server"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/utils"
+	api "google.golang.org/api/compute/v1"
+)
+
+var (
```

Comment:
Should this be `const`?

Reply:
since golang is statically compiled, arrays and slices cannot be constants because arrays and slices are evaluated during runtime.

Reply:
These should not be globals, generateAllTestSetup should pass in pkgManagers and do the appending for pkgTestSetup. If you want to keep pkgTestSetup up here for easy additions that's fine but that's also a bit redundant, just have each of the addCreate* functions use a for loop, why have it defined twice unless you expect different inputs at different times.

Reply:
i wanted to keep the list of pkgmanagers global since i did not want to hide  it in a function. since the list of package managers is static throughout the osconfig package and hence can also be used at other places( in my mind, the list is not just scoped for generating the testsetups, rather its the list of package managers that we support through osconfig). 

but yeah, it makes sense to move the pkgtestsetup to function scope. Let me know if that's ok

Reply:
Having the list global is fine, but the individual addCreate functions should not reference it. Instead generateAllTestSetup should pass it in as an arg for each of the functions. This allows us to run unit tests and refactor in a much more simple matter.